src/github3/orgs.py

```diff
@@ -491,7 +491,7 @@ def add_repository(self, repository, team_id):  # FIXME(jlk): add perms
     @requires_auth
     def blocked_users(
         self, number: int = -1, etag: t.Optional[str] = None
-    ) -> t.Generator[users.ShortUser, None, None]:
+    ) -> t.Iterator[users.ShortUser]:
```

Comment:
I believe `t.Iterator` is likely going to cause issues as `GitHubIterator` has publicly documented additional methods which this type does/will not reveal

Reply:
Do you mean you expect people to be using methods from `t.Generator` or `GitHubIterator`?

Reply:
I mean the latter. The old annotation was certainly also wrkng